docs/src/pages/api/use-field.mdx

```diff
@@ -356,3 +356,36 @@ checked.value; // true or false
 ```
 
 For more information on how you might use the `checked` property, check the [custom checkboxes example](/examples/custom-checkboxes).
+
+## Additional options
+
+To pass additional configuration to your fields you can use more options.
+
+```vue
+<template>
+  <div>
+    <input v-model="value" />
+    <span>{{ errorMessage }}</span>
+  </div>
+</template>
+
+<script setup>
+  import { useField } from 'vee-validate';
+
+  // a simple `company` field with some options passed into it
+  const { value, errorMessage } = useField('company', ..., {
+    label: 'Your company',
+    type: 'text',
+    initialValue: ''
+  });
+</script>
+```
+
+| Generic options   |   | Checkbox/radio options  |
+|-------------------|---|-------------------------|
+| `validateOnMount` |   | `checkedValue`          |
+| `bails `          |   | `uncheckedValue`        |
+| `standalone`      |   | `label`                 |
+| `type`            |   | `validateOnValueUpdate` |
+| `initialValue`    |   | `keepValueOnUnmount`    |
+|                   |   | `syncVModel`            |
```

Comment:
`label`, `validateOnValueUpdate`, `keepValueOnUnmount` and `syncVModel` are not checkbox/radio specific.

Reply:
Apologies for the oversight. I amended my commit and also refactored into a singular column table that will show the `Field options`. `checkedValue` and `uncheckedValue` are prefixed with "(checkbox/radio only)".